schema/transportation/segment.yaml

```diff
@@ -392,6 +417,27 @@ properties:
           description: direction of travel on the target that pertains to the relation
           "$ref": "#/$defs/propertyDefinitions/direction"
       required: [segmentId]
+    compassDirection:
+      description: >-
+        Four cardinal and four ordinal direction
+      type: string
+      enum:
+        - East
+        - West 
+        - North
+        - South 
+        - NorthEast
+        - NorthWest 
+        - SouthEast
+        - SouthWest 
+    signTextType:
+      description: >-
+        Text type for a sign post. Gives general direction as to how to use the sign text 
```

Comment:
It's not clear to me what these enum values mean by your description here. Could you add some additional detail?